src/lib.rs

```diff
@@ -898,6 +897,40 @@ impl Element {
         Ok(self.prop(prop).await?.unwrap())
     }
 
+    /// Find the first matching child element.
```

Comment:
We probably shouldn't refer to "child elements", but rather something like "descendants", as `FindElementElement` is not restricted to immediate children.